Automattic/wp-calypso
# Remove "Profile Links" from /me

Related to https://github.com/Automattic/wp-calypso/issues/94473

## Proposed Changes

This PR removes "Profile Links" from /me.

| before | after |
|--------|--------|
| <img width="1440" alt="Screenshot 2024-12-04 at 17 10 35" src="https://github.com/user-attachments/assets/5b41e31a-ec92-423b-9d9c-771a5ec3028c"> | <img width="1439" alt="Screenshot 2024-12-04 at 17 10 46" src="https://github.com/user-attachments/assets/38945892-cf5a-4f07-9c24-ca9976fb87ea"> | 

## Why are these changes being made?


https://github.com/Automattic/wp-calypso/issues/94473#issuecomment-2516434484

## Testing Instructions



* Go to /me
* Observe no "Profile Links"
* Go to https://gravatar.com/profile/links
* Observe your links exist there if you configured them before

## Pre-merge Checklist
